test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1228,7 +1242,15 @@ def _revert_pbsconf_server(self, vals_to_set):
             if new_pbsconf["PBS_LOG_HIGHRES_TIMESTAMP"] != "1":
                 new_pbsconf["PBS_LOG_HIGHRES_TIMESTAMP"] = "1"
                 restart_pbs = True
-
+            if DAEMON_SERVICE_USER.name == 'root':
+                if "PBS_DAEMON_SERVICE_USER" in new_pbsconf:
+                    del(new_pbsconf['PBS_DAEMON_SERVICE_USER'])
+                    restart_pbs = True
+            elif (new_pbsconf["PBS_DAEMON_SERVICE_USER"] !=
+                  DAEMON_SERVICE_USER.name):
+                new_pbsconf["PBS_DAEMON_SERVICE_USER"] = \
+                    DAEMON_SERVICE_USER.name
+                restart_pbs = True
```

Comment:
What about the case where daemon service user is not provided to benchpress?  Then it is not in pbs.conf by default.  Shouldn't we remove it here?

Reply:
if it's not provided, we do remove it, check line 1246